design/account.go

```diff
@@ -177,6 +177,22 @@ var _ = a.Resource("users", func() {
 		a.Response(d.OK)
 	})
 
+	a.Action("obfuscate", func() {
+		a.Routing(
+			a.PUT("/:id"),
+		)
+		a.Description("Invalidate user details for the given userID.")
+		a.Params(func() {
+			a.Param("id", d.String, "id")
```

Comment:
Can you please change the description and type of param?
```suggestion
			a.Param("userid", d.UUID, "ID of the user whose details are to be invalidated")
```

Reply:
+1

Reply:
the GET/PATCH all use `id` so I'd rather stick with it for consistency

Reply:
I think `id` is fine because it's anyway being used in the context of users. Also, technically it's identityID ;)